### PR TITLE
removed files related to the rearrangement benchmark

### PR DESCRIPTION
**Pull request by ykarmesh**
Removed extra files from the main branch that arent useful for the navigation challenge

### CODE REVIEW

File: README.md
Comment (jacobkrantz) [inline README.md:1]: reference to wrong docker on L148 `fairembodied/habitat-challenge:habitat_rearrangement_2022_base_docker`

File: agents/habitat_baselines_agent.py [hunk 1]
```diff
@@ -308,7 +297,9 @@ def main():
         "--evaluation", type=str, required=True, choices=["local", "remote"]
     )
     parser.add_argument("--model-path", default="", type=str)
-    parser.add_argument("--task", required=True, type=str, choices=["objectnav", "imagenav"])
+    parser.add_argument(
+        "--task", required=True, type=str, choices=["objectnav", "instance_imagenav"]
```
Comment (jacobkrantz) [inline agents/habitat_baselines_agent.py:301]: instance_imagenav -> imagenav

Comment (ykarmesh) [reply agents/habitat_baselines_agent.py:301]: Sorry I think this happened due to some automatic merging of our branches.

File: README.md
Comment (ykarmesh) [reply README.md:1]: done

**Pull request merged**